media-video/wireplumber/wireplumber-9999.ebuild

```diff
@@ -54,6 +54,23 @@ RDEPEND="${DEPEND}"
 
 DOCS=( {NEWS,README}.rst )
 
+presetfile=30-${PN}.preset
+
+src_prepare() {
+	default
+
+	einfo "Generating ${presetfile}"
+	cat > ${presetfile} <<- EOF || die
+		# Start of ${presetfile} from ${P}
+
+		enable pipewire.socket
+		enable pipewire-pulse.socket
```

Comment:
I feel like pipewire (not wireplumber) should install a preset for these, or you should create some separate ebuild that installs them.

Reply:
I guess this doesn't really matter.

Reply:
Another option would be to have PipeWire conditionally installing a higher priority (lower numbered) preset file that specifically disabled pipewire-pulse.socket. This PR was made before it was determined that the current Gentoo default is for user units to be vendor enabled out of the box, so concern was that it would be inappropriate to call `systemctl --global preset foo bar baz` when the actual preset was not provided (and there's little point in PipeWire ebuild enabling only its own units since they are non-functioning without a session manager).